doc/source/user_guide/advanced.rst

```diff
@@ -783,10 +783,10 @@ values **not** in the categories, similarly to how you can reindex **any** panda
 
 .. ipython:: python
 
-   df2.reindex(['a', 'e'])
-   df2.reindex(['a', 'e']).index
-   df2.reindex(pd.Categorical(['a', 'e'], categories=list('abcde')))
-   df2.reindex(pd.Categorical(['a', 'e'], categories=list('abcde'))).index
+   df2.iloc[[0]].reindex(['a', 'e'])
```

Comment:
this is a very confusing example now.  I would just construct a new frame and use that as what I care about as a reader is what does ``df.reindex(....)`` do

Reply:
Agreed, I've changed it.